src/ode_solver/ode_solver_factory.cpp

```diff
@@ -93,6 +123,7 @@ void ODESolverFactory<dim,real,MeshType>::display_error_ode_solver_factory(Param
     else if (ode_solver_type == ODEEnum::rrk_explicit_solver)           solver_string = "rrk_explicit";
     else if (ode_solver_type == ODEEnum::pod_galerkin_solver)           solver_string = "pod_galerkin";
     else if (ode_solver_type == ODEEnum::pod_petrov_galerkin_solver)    solver_string = "pod_petrov_galerkin";
+    else if (ode_solver_type == ODEEnum::hyper_reduced_petrov_galerkin_solver)    solver_string = "hyper_reduced_petrov_galerkin";
```

Comment:
Sorry for being a bit picky, but can you put `solver_string = '...'` on the next line so that it lines up with the other `solver_string`s?

Reply:
done and modified the line below as well to line up

Reply:
Thank you :)